src/index.pug

```diff
@@ -7,6 +7,10 @@ html
       body {
         background-color: #eee !important;
       }
+    script.
```

Comment:
I came to the conclusion that the use `window.require` in script tag is the simplest way to loading `electron`. Because of `DefinePlugin` use common `require` and it does not load electron to the browser.